package.json

```diff
@@ -55,7 +55,7 @@
   "scripts": {
     "build": "grunt build",
     "doc": "grunt doc",
-    "test": "grunt test",
+    "test": "npm run build && grunt test",
```

Comment:
Not accepting this change. When you make your case I could accept `grunt build && grunt test`

Reply:
`npm run build` here guarantees that if a user changes the `build` script, the changes are reflected here. Is your concern that it's NPM-specific? It seemed from the README that this project is developed with NPM.